### DISCUSSION TITLE
Allow customising `created_at`, `updated_at` & `deleted_at` column names

### DISCUSSION THREAD

**Discussion by glerchundi** [aarondl/sqlboiler#1244] [Ideas]
Some of our services in order to accelerate the time to market couple the API field names with the database column names. We also have a strict API design guidelines that are based on [Google's one](https://cloud.google.com/apis/design). In those API design guidelines there are a standard ways of defining properties name: https://google.aip.dev/148.

Taking this into consideration means that in order to implement, for instance, soft deletes, we would like to being able to customise columns that are currently hardcoded within SQLBoiler like `created_at`, `updated_at` & `deleted_at`.

**Comment by glerchundi** [comment]
Friendly pinging @aarondl & @stephenafamo to get a valuable feedback from you two!

**Reply by stephenafamo** [reply]
This has been implemented since `v4.7.0` released on 2021-09-26.

```toml
[auto_columns]
created = "createdAt"
updated = "updatedAt"
deleted = "deletedAt"
```

**Reply by glerchundi** [reply]
Oops! Sorry for the noise @stephenafamo and thanks for taking the time answering. I directly asked because last time I checked it wasn’t possible…I cannot believe it was before that date 😳, time flies! Thanks again!

**Reply by stephenafamo** [reply]
No worries at all. I also think it should be better documented so I understand if it is unclear that it is possible to customize this.
